pandas/core/arrays/datetimelike.py

```diff
@@ -2038,7 +2038,30 @@ def _validate_dtype(cls, values, dtype):
     def freq(self):
         """
         Return the frequency object if it is set, otherwise None.
-        """
+
+        To learn more about the frequency strings, please see `this link
+        <https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases>`__.
```

Comment:
Can you use ``` :ref:`this link<timeseries.offset_aliases>` ```

Reply:
you bet! thanks for this, I found several other places with this same link and using the old format, I'll fix this one here in a new commit, and I'll open a separate PR to address the others.